build.gradle

```diff
@@ -45,8 +45,8 @@ buildscript {
     }
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_6
```

Comment:
wait is AE2 actually Java 6 compatible? How ridiculour.

Reply:
Yup, got some weird ass crashes/errors when we upgraded it to 8

Reply:
Hah, wonder if it'll work this time

Reply:
repo distributed a few test builds on discord. Haven't heard of any crashes yet.

Reply:
They previously worked for some people.  Trying to find the old errors

Reply:
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/5748

https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/commit/42c60485e9b686d76360a3e619e69d2b360e4e0c

Reply:
Make sure to test this on Thermos before merging w/ java 8 compat